js/app/modules/card/knowledgeDocument.js

```diff
@@ -82,6 +82,12 @@ const KnowledgeDocument = new Module.Class({
         'next-card': GObject.ParamSpec.object('next-card',
             'Next Card', 'Next Card',
             GObject.ParamFlags.READWRITE | GObject.ParamFlags.CONSTRUCT_ONLY, Gtk.Widget),
+        /**
+         * Property: nav-content
+         */
+        'nav-content': GObject.ParamSpec.object('nav-content',
```

Comment:
@ptomato I wonder if we are still interested in having the previous-card? I think we stopped using it and maybe we can get rid of it. We could also get rid completely of "*-card" slots and just having this more generic "content" slot, where we would put the next article card.

What do you think?